client/src/reviewService.jsx

```diff
@@ -19,24 +19,26 @@ class ReviewService extends React.Component {
     this.setReviewsFilteredBySearch = this.setReviewsFilteredBySearch.bind(this);
     this.setReviewsFilteredByTier = this.setReviewsFilteredByTier.bind(this);
     this.setReviewsFilteredBySearchAndTier = this.setReviewsFilteredBySearchAndTier.bind(this);
-
+    this.showTwelveMoreReviews = this.showTwelveMoreReviews.bind(this);
     this.state = {
-      courseId: null,
+      courseId: Number(querystring.parse(window.location.search)['?courseId']),
       totalReviews: null,
       currentSearchTerm: null,
       reviewsBySearch: null,
       currentTier: null,
       reviewsByTier: null,
       reviewsBySearchAndTier: null,
       featuredReview: null,
-      ratings: null
+      ratings: null,
+      displayedReviews: null
     };
+    this.getReviews(this.state.courseId);
   }
 
-  componentDidMount() {
-    let courseId = Number(querystring.parse(window.location.search)['?courseId']);
-    this.getReviews(courseId);
-  }
+  // componentDidMount() {
+  //   let courseId = Number(querystring.parse(window.location.search)['?courseId']);
+  //   this.getReviews(courseId);
+  // }
 
```

Comment:
I think this can be safely deleted now, right?

Reply:
just did! I was keeping it around just in case.